src/python/pants/option/option_types.py

```diff
@@ -761,12 +761,12 @@ def _convert_(self, val: Any) -> dict[str, _ValueT]:
 class SkipOption(BoolOption[bool]):
     """A --skip option (for an invocable tool)."""
 
-    def __new__(cls, goal: str, *other_goals: str):
+    def __new__(cls, goal: str, *other_goals: str, flag_name: str | None = None):
```

Comment:
Nit, could simplify:

```suggestion
    def __new__(cls, goal: str, *other_goals: str, flag_name: str = "--skip"):
```

Reply:
Fixed! :)